src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -1543,4 +1592,29 @@ private PoolStats getNoTimeoutClientConnectionPoolStats(final DefaultDockerClien
     return ((PoolingHttpClientConnectionManager) client.getNoTimeoutClient().getConfiguration()
         .getProperty(ApacheClientProperties.CONNECTION_MANAGER)).getTotalStats();
   }
+
+  private String createNoTimeoutContainer() throws Exception {
```

Comment:
Maybe we can name this `createSleepingContainer` to indicate what it does instead of what test it's for?